test/functional/test-ads-config.js

```diff
@@ -18,6 +18,12 @@ import {adConfig} from '../../ads/_config';
 
 describe('test-ads-config', () => {
 
+  it('should have all ad networks configured', () => {
+    window.ampTestRuntimeConfig.adTypes.forEach(adType => {
+      expect(adConfig, `Missing config for [${adType}]`).to.contain.key(adType);
+    });
+  });
```

Comment:
I removed the check for `_ping_` because I think that's not important at all, itself is for test only.